netutils/netinit/netinit_associate.c

```diff
@@ -60,7 +60,7 @@
 int netinit_associate(FAR const char *ifname)
 {
   struct wpa_wconfig_s conf;
-  int ret;
+  int ret = -EINVAL;
```

Comment:
This is causing the build to fail. errno.h is not included.

Reply:
Done